pkg/scheduler/service.go

```diff
@@ -19,14 +19,14 @@ type (
 		dispatcher dispatcher
 
 		// Read & write locking
-		l *sync.RWMutex
+		l sync.RWMutex
 
 		// Simple chan to control if service is running or not
 		ticker *time.Ticker
 	}
 
 	dispatcher interface {
```

Comment:
Should the interface be also renamed then?

Reply:
sorry, what interface?